system-addon/content-src/styles/_theme.scss

```diff
@@ -44,6 +44,8 @@
   // Cards
   --newtab-card-background-color: $white;
   --newtab-card-active-outline-color: $grey-30;
+  --newtab-card-image-background-color: $grey-10;
```

Comment:
Would be good to check if the background for the image just be the background for the whole card. I.e., white for default and $grey-90 for dark.